src/Glpi/System/RequirementsManager.php

```diff
@@ -200,6 +197,7 @@ function ($directory) {
         );
 
         if ($db instanceof \DBmysql) {
+            $requirements[] = new DbEngine($db);
```

Comment:
It should not be moved here. The display order in the UI correspond to the declaration order here.